db/order_item_seeds.csv

```diff
@@ -0,0 +1,26 @@
+quantity,product_id,order_id
+,7,12
+,5,14
+,19,14
+,18,8
+,3,10
+,4,9
+,15,11
+,11,9
+,4,18
+,18,17
+,18,22
+,8,2
+,16,12
+,19,8
+,9,8
+,14,9
+,22,2
+,6,10
+,3,13
+,13,4
+,15,3
+,9,4
+,20,8
+,5,22
+,1,6
```

Comment:
Are we missing all the quantities here or it should be empty?

Reply:
I kept the quantity empty because I didn't know how to populate that field such that it wouldn't exceed the "stock" value. I'll work on that bit.